openmc/universe.py

```diff
@@ -360,18 +361,18 @@ def plot(self, origin=(0., 0., 0.), width=(1., 1.), pixels=(200, 200),
 
             # Create a figure sized such that the size of the axes within
             # exactly matches the number of pixels specified
-            if kwargs.get("ax") is None:
+            if axes is None:
                 px = 1/plt.rcParams['figure.dpi']
-                fig, ax = plt.subplots()
+                fig, axes = plt.subplots()
                 params = fig.subplotpars
                 width = pixels[0]*px/(params.right - params.left)
                 height = pixels[0]*px/(params.top - params.bottom)
                 fig.set_size_inches(width, height)
             else:
-                ax = kwargs.pop("ax")
+                pass
```

Comment:
```suggestion
```

Just wondering if this ```pass``` and the above ```else:``` are needed as they don't appear to do anything